### PR TITLE
WFCORE-1799 - Property separator not appended after argument value if…

### PR DESCRIPTION
**Pull request by spyrkob**
… argument completer uses non-zero offset

JIRA: https://issues.jboss.org/browse/WFCORE-1799

### CODE REVIEW

**Comment by spyrkob** [comment]
retest this please

**Comment by bstansberry** [comment]
@jfdenise Please approve

File: cli/src/main/java/org/jboss/as/cli/operation/OperationRequestCompleter.java [hunk 1]
```diff
@@ -492,6 +492,19 @@ protected int complete(CommandContext ctx, ParsedCommandLine parsedCmd, Operatio
...
     }
 
+    private boolean suggestionEqualsUserEntry(List<String> candidates, String chunk, int suggestionOffset) {
+        if (chunk == null) {
+            return false;
+        }
+
+        if (suggestionOffset > 0) {
+            // user entry before suggestionOffset is always the same - compare only part after offset
+            return candidates.size() == 1 && chunk.substring(suggestionOffset).equals(candidates.get(0));
```
Comment (jfdenise) [inline cli/src/main/java/org/jboss/as/cli/operation/OperationRequestCompleter.java]: Hi @spyrkob,
excellent! Perhaps you could move the 2 candidates.size() == 1 to a single checkline 496.
if(chunk == null || candidates.size() == 1)

**Comment by spyrkob** [comment]
@jfdenise thanks, I changed the condition as you suggested.

**Comment by jfdenise** [comment]
@bstansberry, that is good to merge.

**Pull request merged**